index.html

```diff
@@ -76,78 +76,45 @@ <h2 class="primary-color supporting-headline">
-                <li>html</li>
-              </ul>
-              <button type="button" class="btn">See Project</button>
+          <div class="project-intro">
+            <div class="project-title">
+              <h2 class="modal-header">title header</h2>
             </div>
-          </div>
-          <div class="card project-one">
-            <div class="card-text">
-              <h3>Multi-Post Stories Gain+Glory</h3>
-              <ul class="work-catagories">
-                <li>Ruby on rails</li>
-                <li>css</li>
-                <li>JavScript</li>
-                <li>html</li>
-              </ul>
-              <button type="button" class="btn">See Project</button>
+            <div class="project-btn desktop-view-btn">
+              <a href="" class="desktop-see-live-btn btn">
+                See live
+                <img src="./icons/SeeProjectIcon.svg" alt="See live" />
+              </a>
+              <a href="" class="desktop-see-source-btn btn">
+                See Source <img src="./icons/Vector.svg" alt="" />
+              </a>
             </div>
           </div>
-          <div class="card project-one">
-            <div class="card-text">
-              <h3>Multi-Post Stories Gain+Glory</h3>
-              <ul class="work-catagories">
-                <li>Ruby on rails</li>
-                <li>css</li>
-                <li>JavScript</li>
-                <li>html</li>
-              </ul>
-              <button type="button" class="btn">See Project</button>
-            </div>
+          <div class="project-codekit">
+            <div class="tags"></div>
           </div>
-          <div class="card project-one">
-            <div class="card-text">
-              <h3>Multi-Post Stories Gain+Glory</h3>
-              <ul class="work-catagories">
-                <li>Ruby on rails</li>
-                <li>css</li>
-                <li>JavScript</li>
-                <li>html</li>
-              </ul>
-              <button type="button" class="btn">See Project</button>
-            </div>
+          <div class="project-description">
+            <p class="modal-project-description"></p>
           </div>
-          <div class="card project-one">
-            <div class="card-text">
-              <h3>Multi-Post Stories Gain+Glory</h3>
-              <ul class="work-catagories">
-                <li>Ruby on rails</li>
-                <li>css</li>
-                <li>JavScript</li>
-                <li>html</li>
-              </ul>
-              <button type="button" class="btn">See Project</button>
-            </div>
+          <div class="project-btn mobile-view-btn">
+            <a href="" class="moblie-see-live-btn btn">
+              See live
+              <img src="./icons/SeeProjectIcon.svg" alt="See live" />
+            </a>
+            <a href="" class="moblie-see-source-btn btn">
+              See Source <img src="./icons/Vector.svg" alt="" />
+            </a>
```

Comment:
- [ ] Kindly add the section of the entire work dynamically from js by removing the existing code from HTML and creating it dynamically from js.

Reply:
@sja-thedude I have removed all the HTML code for the entire work section and created it dynamically from the js file. Kindly Please check it(on the commit e0ab089f7c51512028f60c32de5d185b47653a84 or you can see it in the index.js file from line number 120 to 141 ). But I have created the popup window using my HTML and appended the necessary data from the js file dynamically. Is any problem with that?